src/Extracting/Strategies/GetFromInlineValidatorBase.php

```diff
@@ -90,7 +90,7 @@ enum_exists($enum) && method_exists($enum, 'tryFrom')
                         // $case->value only exists on BackedEnums, not UnitEnums
                         // method_exists($enum, 'tryFrom') implies $enum instanceof BackedEnum
                         // @phpstan-ignore-next-line
-                        $rulesList[] = 'in:' . implode(',', array_map(fn ($case) => $case->value, $enum::cases()));
+                        $rulesList[] = 'in:' . implode(',', array_map(fn ($case) => "$case->value ($case->name)", $enum::cases()));
```

Comment:
I'm deleting this line because it's not needed. The rulesList is the actual list of rules that Laravel supports, and changing this from `in:1,2,3` to `in:1 (MALE),2 (FEMALE),3 (UNDEFINED)` will probably break things.